request-controller/src/main/java/org/wildfly/extension/requestcontroller/RequestController.java

```diff
@@ -59,7 +59,7 @@
  */
 public class RequestController implements Service<RequestController>, ServerActivity {
 
-    public static final ServiceName SERVICE_NAME = ServiceName.JBOSS.append("server", "global-request-controller");
+    public static final ServiceName SERVICE_NAME = RequestControllerRootDefinition.REQUEST_CONTROLLER_CAPABILITY.getCapabilityServiceName();
```

Comment:
In this case it's probably okay, but generally service names are retrieved from the [`RuntimeCapabilityRegistry`](https://github.com/wildfly/wildfly-core/blob/master/controller/src/main/java/org/jboss/as/controller/capability/registry/RuntimeCapabilityRegistry.java#L47-47). This is implemented in the [`ModelControllerImpl`](https://github.com/wildfly/wildfly-core/blob/master/controller/src/main/java/org/jboss/as/controller/ModelControllerImpl.java#L1420-L1423). Currently I don't think the `CapabilityContext` will be an issue with this use-case, but I don't know where Brian is going with it.